### ISSUE TITLE
Serial and Batch Number Convention for UID

### ISSUE THREAD

**Issue by EngineerRandyN** [EngineerRandyN/MRPHero#92]
When a product is released into the market, it needs to be assigned a **serial and batch number to identify its UID.**

The serial number is the product's position in manufacturing of a group. The batch number is the group # it was manufactured in.

For example. Serial #001 and Lot #0001 would indicate the first product of the first batch. Serial #04 and Lot #05 means the fourth manufactured product in the fifth batch.

The UID means **Unique Identifier** and that is indicated by the Serial and Batch number. I propose we do something like this: If we have a product with Serial #004 and Batch #0003, then its **UID** should be **S004-B0003**

Long story short. Serial number is the product's position in the manufacturing batch. We create the UID by combining the Serial number and its Lot number.
